packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -54,10 +50,28 @@ class CodeSizeController {
     _diffRoot.value = newRoot;
   }
 
+  ValueListenable<DevToolsJsonFile> get oldDiffSnapshotJsonFile {
```

Comment:
you can still leave this as an inline getter (using `=>`). here and elsewhere. I don't think putting it in brackets adds any value as it is just a single return statement

Reply:
The reason why I added this was because the entire line didn't fit on a single line when using an inline getter. Flutter style guide suggests the following:
```
// BAD:
String capitalize(String s) =>
  '${s[0].toUpperCase()}${s.substring(1)}';

// GOOD:
String capitalize(String s) => '${s[0].toUpperCase()}${s.substring(1)}';

String capitalize(String s) {
  return '${s[0].toUpperCase()}${s.substring(1)}';
}
``` 
But I'm not sure if our repo adheres strictly to the style guide so I can change it back if we prefer the other way.